src/dropin-model.js

```diff
@@ -137,6 +137,10 @@ DropinModel.prototype.confirmPaymentMethodDeletion = function (paymentMethod) {
   this._emit('confirmPaymentMethodDeletion', paymentMethod);
 };
 
+DropinModel.prototype.shouldExpandPaymentOptions = function () {
+  return Boolean(this.merchantConfiguration.expandOtherWaysToPay && this.getPaymentMethods().length > 0);
```

Comment:
will there ever be a situation where only cards are supported, so `this.getPaymentMethods.length = 1` and we _don't_ want to expand other ways to pay? i.e. should this be kicked up to `> 1`?

Reply:
Good call. This bit is fine as is, but a check for `this.supportedPaymentOptions.length > 1` does need to be added.